test/support/user_controller_annotated.ex

```diff
@@ -27,6 +27,7 @@ defmodule OpenApiSpexTest.UserControllerAnnotated do
          },
          unauthorized: Unauthorized.response(),
          not_found: NotFound.response(),
+         unprocessable_entity: %Reference{"$ref": "#/components/responses/unprocessable_entity"},
```

Comment:
Let's also update `examples/phoenix_app` showing how this can be used.

Reply:
Does this work? https://github.com/open-api-spex/open_api_spex/pull/532/commits/a3824e4286c417657ca94f795c117dd15530201e